NumberBaseball/NumberBaseball/main.swift

```diff
@@ -3,15 +3,56 @@
 //  Created by BaekGom,barrd.
 //  Copyright © yagom academy. All rights reserved.
 //
+import Foundation
 
 var repeatCheck = true
 let countNumber = 3
 let randomNumberRange = 1...9
-let arrayRange = 0...2
+let arrayExtent = 0...2
 let gameOverStrikeCount = 3
 let gameOverTryCount = 0
 
-func generateThreeNonOverlappingRandomNumbers() -> Set<Int> {
+func selectMenu() {
+    print("1. 게임시작")
+    print("2. 게임종료")
+    print("원하는 기능을 선택해주세요 : ", terminator: "")
+    let userSelectMenu = readLine()
+    
+    if userSelectMenu == "1" {
+        repeatCheck = true
+        gameStart()
+        selectMenu()
+    } else if userSelectMenu == "2" {
+        print("", terminator: "")
+    } else {
+        print("입력이 잘못되었습니다")
+        selectMenu()
+    }
+}
+
+func userInputNumbers() -> String {
+    print("숫자 3개를 띄어쓰기로 구분하여 입력해주세요")
+    print("중복 숫자는 허용하지 않습니다.")
+    print("입력 : ", terminator: "")
+    let userInput = readLine() ?? ""
+    
+    if confirmInputNumbers(InputNumbers: userInput) {
+        return userInput
+    }else{
+        print("입력이 잘못되었습니다")
+        return userInputNumbers()
+    }
```

Comment:
컨벤션을 통일해주세요!! 하지만 99%는 다음과 같이 사용합니다
```suggestion
    } else {
        print("입력이 잘못되었습니다")
        return userInputNumbers()
    }
```

Reply:
확인작업을 해도 하나씩 고칠 점이 있네요.  
더 꼼꼼히 살펴보겠습니다.